misc/acrn-config/xmls/config-xmls/apl-mrb/sdc_launch_1uos_aaag.xml

```diff
@@ -5,7 +5,7 @@
 	<mem_size desc="UOS memory size in MByte">2048</mem_size>
 	<gvt_args desc="GVT argument for the vm">64 448 8</gvt_args>
 	<vbootloader desc="virtual bootloader method" readonly="true">ovmf</vbootloader>
-	<console_type desc="UOS console type">virtio-console(hvc0)</console_type>
+	<vuart0 desc="vUART0 which emulated by device model">Disable</vuart0>
```

Comment:
Please also describe vuart in commit message.

Reply:
OK, got it.